hrui/hrui.php

```diff
@@ -934,3 +936,113 @@ function _hrui_menuSetActive($isActive) {
   CRM_Core_DAO::executeQuery("UPDATE civicrm_navigation SET is_active = {$isActive} WHERE name = 'import_custom_fields'");
   CRM_Core_BAO_Navigation::resetNavigation();
 }
+
+function _getMenuKeyMax($menuArray) {
+  $max = array(max(array_keys($menuArray)));
+  foreach($menuArray as $v) {
+    if (!empty($v['child'])) {
+      $max[] = _getMenuKeyMax($v['child']);
+    }
+  }
+  return max($max);
+}
+
+function civicrm_civicrm_navigationMenu(&$menu) {
```

Comment:
Does this work? The convention is to use the extension name as a prefix to the hook, so it should be `hrui_civicrm_navigtionMenu`. Also please run `civilint` when you change this before committing. Hooks should have a doc-block that says "Implements hook_xxx". You should also add the documentation block for the parameters.